src/legacy/core_plugins/kbn_vislib_vis_types/public/controls/point_series/series.js

```diff
@@ -1,102 +0,0 @@
-/*
- * Licensed to Elasticsearch B.V. under one or more contributor
- * license agreements. See the NOTICE file distributed with
- * this work for additional information regarding copyright
- * ownership. Elasticsearch B.V. licenses this file to you under
- * the Apache License, Version 2.0 (the "License"); you may
- * not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *    http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing,
- * software distributed under the License is distributed on an
- * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
- * KIND, either express or implied.  See the License for the
- * specific language governing permissions and limitations
- * under the License.
- */
-
-import _ from 'lodash';
-import { uiModules } from 'ui/modules';
-import vislibSeriesTemplate from './series.html';
-import { safeMakeLabel } from 'ui/agg_types/agg_utils';
-
-const module = uiModules.get('kibana');
-
-module.directive('vislibSeries', function () {
-  return {
-    restrict: 'E',
-    template: vislibSeriesTemplate,
-    replace: true,
-    link: function ($scope) {
-      function makeSerie(id, label) {
```

Comment:
**note:** Moved to https://github.com/elastic/kibana/pull/43772/files?file-filters%5B%5D=.ts&hide-deleted-files=true#diff-168caa277cae11f70ad28d6eb73f43e0R24